Running bob clean targets in non-bob folder panics

```
daniel@benchkram-dketterer Downloads ➜ bob clean targets
Unable to aggregate bob file: Could not find a bob.yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x1273a16]

goroutine 1 [running]:
github.com/benchkram/bob/cli.runCleanTargets()
	/home/runner/work/bob/bob/cli/cmd_clean.go:59 +0x96
github.com/benchkram/bob/cli.glob..func12(0x21f8820?, {0x15bfcba?, 0x0?, 0x0?})
	/home/runner/work/bob/bob/cli/cmd_clean.go:48 +0x17
github.com/spf13/cobra.(*Command).execute(0x21f8820, {0x2245150, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0x21f7420)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/benchkram/bob/cli.Execute()
	/home/runner/work/bob/bob/cli/cli.go:27 +0x4c
main.main()
	/home/runner/work/bob/bob/main.go:14 +0x4c
```

--- a/cli/cmd_clean.go
+++ b/cli/cmd_clean.go
@@ -2,17 +2,20 @@ package cli
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/benchkram/bob/bob"
 	"github.com/benchkram/bob/pkg/boblog"
+	"github.com/benchkram/bob/pkg/usererror"
+	"github.com/benchkram/errz"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 var cleanCmd = &cobra.Command{
 	Use:   "clean",
 	Short: "Clean buildinfo and artifacts",
-	//Args:  cobra.ExactArgs(1),
-	Long: ``,
+	Long:  ``,
 	Run: func(cmd *cobra.Command, args []string) {
 		_ = cmd.Help()
 	},
@@ -20,9 +23,10 @@ var cleanCmd = &cobra.Command{
 
 var cleanSystemCmd = &cobra.Command{
 	Use:   "system",
-	Short: "Clean targets",
-	//Args:  cobra.ExactArgs(1),
-	Long: ``,
+	Short: "Remove buildinfo and local cache",
+	Long: `Remove all entries from 
+  ~/.bobcache/buildinfo 
+  ~/.bobcache/artifacts`,
 	Run: func(cmd *cobra.Command, args []string) {
 		runCleanSystem()
 	},
@@ -41,9 +45,8 @@ func runCleanSystem() {
 
 var cleanTargetsCmd = &cobra.Command{
 	Use:   "targets",
-	Short: "Clean targets",
-	//Args:  cobra.ExactArgs(1),
-	Long: ``,
+	Short: "Remove targets declared by the current project",
+	Long:  `Remove filesystem targets declared by the current project, docker targets are not removed`,
 	Run: func(cmd *cobra.Command, args []string) {
 		runCleanTargets()
 	},
@@ -54,7 +57,14 @@ func runCleanTargets() {
 	boblog.Log.Error(err, "Unable to initialise bob")
 
 	ag, err := b.Aggregate()
-	boblog.Log.Error(err, "Unable to aggregate bob file")
+	if err != nil {
+		if errors.As(err, &usererror.Err) {
+			boblog.Log.UserError(err)
+		} else {
+			errz.Fatal(err)
+		}
+		os.Exit(1)
+	}
 
 	for _, t := range ag.BTasks {
 		err := t.Clean(true)

